src/analysis.c

```diff
@@ -65,20 +119,47 @@ int mii_analysis_run(const char* modfile, int modtype, char*** bins_out, int* nu
     return 0;
 }
 
+#if MII_ENABLE_LUA
+/*
+ * run a modulefile's code in a lua sandbox
+ */
+char** _mii_analysis_lua_run(lua_State* lua_state, const char* code) {
+    /* execute modulefile */
+    lua_getglobal(lua_state, "sandbox_run");
+    lua_pushstring(lua_state, code);
+    lua_call(lua_state, 1, 1);
```

Comment:
I changed the `lua_call` to a `lua_pcall` to handle all errors in the Lua sandbox, since it is probably the easiest to understand and maintain and also probably the fastest. I'm open to changing it so the errors in the modulefiles and the Lua sandbox function are handled separately.

For now, I used the `mii_error` function to display the errors, but I could change it to use `mii_warn` instead.